parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetReader.java

```diff
@@ -51,6 +51,7 @@
   private final Iterator<Footer> footersIterator;
   private final Filter filter;
 
+  private String createdBy;
```

Comment:
does this have to be stored in an instance variable? seems like it could be a local variable?